core-processor/src/main/java/io/micronaut/inject/beans/visitor/BeanIntrospectionWriter.java

```diff
@@ -722,7 +722,8 @@ private void writeIntrospectionClass(ClassWriterOutputVisitor classWriterOutputV
         dispatchWriter.buildGetTargetMethodByIndex(classWriter);
         buildFindIndexedProperty(classWriter);
         buildGetIndexedProperties(classWriter);
-        boolean hasBuilder = annotationMetadata != null && annotationMetadata.isPresent(Introspected.class, "builder");
+        boolean hasBuilder = annotationMetadata != null &&
+            (annotationMetadata.isPresent(Introspected.class, "builder") || annotationMetadata.hasDeclaredAnnotation("lombok.Builder"));
```

Comment:
Can you add an annotation remapper to avoid referencing the Lombok here?

Reply:
but also are we sure we want to generate an instrospection for all Lombok annotated builders?

Reply:
Then we can add a visitor that will only remap it if the builder annotation is present

Reply:
so I move the reference to Lombok from one place to another?